### DISCUSSION TITLE
2021/08/shiny-in-production/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#97] [General]
# 2021/08/shiny-in-production/

https://xiangyun.rbind.io/2021/08/shiny-in-production/

**Comment by botsp** [comment]
前辈，请问怎样R package与SAS macro的区别，似乎都只是制作一个function?

**Reply by botsp** [reply]
在数据与统计处理领域

**Reply by XiangyunHuang** [reply]
抱歉，今天才回过头来看我的讨论区，发现很多没回复的。SAS 我没有用过，不了解。在数据与统计处理领域，我编程很多时候都是写函数，基本没有使用过面向对象编程。

**Reply by botsp** [reply]
谢谢、我自己反复理解了下，感觉还是似是而非，可能还是写的太少了。
